### PR TITLE
set initial scale to 1

### PR DESCRIPTION
**Pull request by MayKikuchi**
setting the initial scale of the zoom to 1 because it messes with the island sizing if it starts out zoomed out :<

### CODE REVIEW

File: surabaya-client/src/components/RightSide/index.tsx [hunk 1]
```diff
@@ -31,15 +31,17 @@ const RightSide: React.FC<RightSideProps> = (props: RightSideProps) => {
...
+  // const initialScale = Math.min(
+  //   1,
+  //   Math.min(
+  //     (window.innerWidth - 300) / javaProject.width,
+  //     window.innerHeight / javaProject.height
+  //   )
+  // );
   const initialZoomValue = {
-    scale: initialScale,
+    scale: 1,
```
Comment (algobot76) [inline surabaya-client/src/components/RightSide/index.tsx:44]: Please make `1` a constant.

File: surabaya-client/src/components/TooltipSquare/index.tsx [hunk 2]
```diff
@@ -45,7 +45,7 @@ const TooltipSquare: React.FC<TooltipSquareProps> = (
   const avgClusterWidth = totalClusterWidth / clusterNum;
   const numberOfClusters = fileData.classes.length;
   const columns = getNumColumnsForSquare(numberOfClusters);
-  const width = avgClusterWidth * columns * 1.2;
+  const width = avgClusterWidth * columns * 1.5;
```
Comment (algobot76) [inline surabaya-client/src/components/TooltipSquare/index.tsx:48]: Please make `1.5` a constant.

File: surabaya-client/src/util/helpers.ts [hunk 3]
```diff
@@ -42,8 +42,7 @@ export function getTooltipWidth(fileData: JavaFile): number {
   const avgClusterWidth = totalClusterWidth / clusterNum;
   const numberOfClusters = fileData.classes.length;
   const columns = getNumColumnsForSquare(numberOfClusters);
-  const width = avgClusterWidth * columns * 1.2;
-  return width;
+  return avgClusterWidth * columns * 1.5;
```
Comment (algobot76) [inline surabaya-client/src/util/helpers.ts:45]: Same here.

File: surabaya-client/src/components/RightSide/index.tsx (hunk 1 above)
Comment (MayKikuchi) [reply surabaya-client/src/components/RightSide/index.tsx:44]: I don't think there's any reason to make a scale of 1 a constant lol, it just meanst 1:1 ratio

**Pull request merged**